tests/Baseline/hilti.rt.profiler/output

```diff
@@ -0,0 +1,5 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+#name count avg-% total-%
+hilti/func/Foo::fibo 2
```

Comment:
Not a very strong opinion, but I personally find the removal of recursive calls confusing. If I was implementing a cache as a user profiling would give me a "wrong" answer.

Reply:
Hmm. it's primarily because the time accounting doesn't work. I think we could still update the count, let me see.

Reply:
That works.